lib/tessel/deployment/javascript.js

```diff
@@ -68,6 +68,17 @@ exportables.preBundle = function(opts) {
   return exportables.resolveBinaryModules(opts);
 };
 
+// exportables.preRun = function(tessel) {};
+
+exportables.postRun = function(tessel, remote) {
```

Comment:
I'm not sure that holds up very well, considering:

- There is no such thing as a "remote process" when `preRun` is called, but would also be irrelevant to that operation.
- `preBundle` and `tarBundle` are about files and directories, not Tessel or "remote process". 


The best that I think can be done is:

- `preBundle(options)` 
- `tarBundle(options)` 
- `preRun(tessel, options)`
- `postRun(tessel, options)` (`options` will contain `remoteProcess`)